src/install.js

```diff
@@ -85,10 +85,21 @@ const versionEndpoint = url.parse('https://sapui5.hana.ondemand.com/resources/sa
 const downloadEndpoint = url.parse('https://tools.hana.ondemand.com/additional/');
 
 (async () => {
+  let latestVersionURL
+  // Check if a specific Version of UI5 is needed..
+  // we are in node_modules/sapui5-runtime/src and need to go down to project using our module
+  const packageJson = fs.readJsonSync(path.join(__dirname, '../../../package.json'), { throws: false })
+
+  if (packageJson && packageJson.sapui5RuntimeVersion) {
+    latestVersionURL = url.resolve(downloadEndpoint.href, `sapui5-rt-${packageJson.sapui5RuntimeVersion}.zip`)
+  }
+
   prepareFileSystem(libDir, downloadDir)
 
   try {
-    const latestVersionURL = await determineLatestVersionURL(versionEndpoint, downloadEndpoint)
+    if (!latestVersionURL) {
+      latestVersionURL = await determineLatestVersionURL(versionEndpoint, downloadEndpoint)
+    }
```

Comment:
With my suggested changes, this could be simplified to:
```javascript
if (!config.version) {
  latestVersionURL = await determineLatestVersionURL(versionEndpoint, downloadEndpoint)
} else {
  latestVersionURL = url.resolve(downloadEndpoint.href, `sapui5-rt-${config.version}.zip`)
}
```